notebooks/1.0-vse-eda-rooms.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<ul><li>It's better to load datasets with all columns and filters in the notebook. The reason is that the rest of the team didn't have the file 'rooms.csv' that you're loading.</li><li>It's better to use relative paths because the absolute path is different in each computer</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>By relative paths, you mean a path in our repository?</p>

Reply:
<p>look this link: <a href="https://www.computerhope.com/issues/ch001708.htm" target="_blank">https://www.computerhope.com/issues/ch001708.htm</a></p>

Reply:
<p>Yes, the repository is useful to all people that are developing the project save files in the same relative location. So, you need to put the CSV file in data/raw and access with a relative path, to work on all computers.</p>